docs/components/shell/TableOfContents.tsx

```diff
@@ -0,0 +1,47 @@
+export const TableOfContents = ({ content }) => {
```

Comment:
Let's turn on `strict: true` in `tsconfig.json` which will start asking you to type things like `{content}`